tests/validation/cattlevalidationtest/core/test_rancher_compose_commands.py

```diff
@@ -0,0 +1,620 @@
+from common_fixtures import *  # NOQA
+
+TEST_SERVICE_OPT_IMAGE = 'ibuildthecloud/helloworld'
+TEST_SERVICE_OPT_IMAGE_LATEST = TEST_SERVICE_OPT_IMAGE + ':latest'
+TEST_SERVICE_OPT_IMAGE_UUID = 'docker:' + TEST_SERVICE_OPT_IMAGE_LATEST
+LB_IMAGE_UUID = "docker:sangeetha/testlbsd:latest"
+RCCOMMANDS_SUBDIR = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                 'resources/rccmds')
+logger = logging.getLogger(__name__)
+
+
+if_compose_data_files = pytest.mark.skipif(
+    not os.path.isdir(RCCOMMANDS_SUBDIR),
+    reason='Rancher compose files directory location not set/does not Exist')
+
+
+@if_compose_data_files
```

Comment:
Can you add comments to include the information about the command this is being verified